CONTRIBUTING.md

```diff
@@ -210,6 +211,17 @@ Python file with `jupyter nbconvert notebook_utils.ipynb --TagRemovePreprocessor
 Add a "hide" tag to any demo cells (from the right side gear sidebar) to prevent these cells from being added to the script.
 
 
+### Interactive inference with Gradio
+To enhance the functionality of your notebook, it is recommended to include an interactive model inference interface at the end. We recommend using [Gradio](https://gradio.app) for this purpose.
+
+Here are some guidelines to follow:
+
+ - Install the latest version of Gradio by running `pip install gradio`.
```

Comment:
```suggestion
 - Install the latest version of Gradio by running `pip -q install gradio`.
```
BTW, it speaks nothing about latest version (if user aleady have it, it will not be updated)